index.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+module.exports = () => ({
+    transform: {
+        '\\.js$': require.resolve('babel-jest'),
+        '\\.(png|jpg|jpeg|gif|webp|ico)$': require.resolve('jest-file'),
+        '\\.(eot|ttf|woff|woff2|otf)$': require.resolve('jest-file'),
+        '\\.(mp3|flac|wav|aac|ogg|oga|mp4|m4a|webm|ogv)$': require.resolve('jest-file'),
+        '\\.(svg)$': require.resolve('jest-file'),
+    },
+    moduleNameMapper: {
+        '\\.css$': require.resolve('identity-obj-proxy'),
+    },
+    collectCoverage: true,
```

Comment:
I don't think this is a good default. I think it's a good default for a CI but many times, when in a project, I rather have the tests run without coverage while I'm still developing them.

Since this property is set here, if I want to not have coverage in my projects, I have to run this config and then mutate it's result. If this property isn't set here, I can run `jest --coverage` when I want coverage and `jest --watch` or something else when I don't want coverage. Without changing any configuration files.

@satazor @threequartersjohn what do you think?

Reply:
I understand, though people will neglect coverage and the coverage thresholds won't be reached if we disable it by default.

One solution would be to enable by default on CI's, but still it means that people would only see the problem when they open a PR.

Reply:
Personally, I would more quickly suggest to a developer that they have coverage always on than not. 

And, this being a boilerplate, I think it would be better to be stringent about guaranteeing the developer is aware of coverage.

I would not consider having to personalize the output to reduce information logging a bad thing or too much effort, and would rather have that than moving that effort into changing it to log coverage for developers who want it always on.